net-mgmt/opnarplog/src/opnsense/service/conf/actions.d/actions_opnarplog.conf

```diff
@@ -0,0 +1,29 @@
+[start]
+command:/usr/local/bin/bash /usr/local/etc/rc.d/opnarplog start
```

Comment:
You could use the service commands here:
```
[start]
command:service opnarplog start
parameters:
type:script
message:starting OPNarplog
```
After changing it restart configd:

`service configd restart`

Then try it out with:

`configctl opnarplog start`

Reply:
Nice, thank you! That also helped me with the 1st problem. `/usr/local/bin/bash` was not available on the system and therefore it did not work.

Reply:
/bin/sh is there, if you need bash you have to set PLUGIN_DEPENDS bash in Makefile

Reply:
Please do not add bash scripts. As a general rule we do kot want to maintain them.

Reply:
Yes, I already removed it, so there are no dependencies left.